modxcms/revolution
# Remove the scrolling overflow from the modx-browser panel

### What does it do?
Remove margins from the container and add them to the border layout

### Why is it needed?
Remove the scrolling overflow from the modx-browser panel

Before:

![Bildschirmfoto 2022-10-10 um 14 09 38](https://user-images.githubusercontent.com/148371/194863384-32eef754-882c-44ba-8b0a-cfb8ec7e597e.png)

After:

![Bildschirmfoto 2022-10-10 um 14 11 20](https://user-images.githubusercontent.com/148371/194863692-cae2ead7-0905-4016-8992-b1e41044765b.png)

### How to test
Add the changes and check the browser panel in the MODX media browser CMP. It currently has to be tested with responsive sizes. The CSS can perhaps be simplified there. 

Maybe it is better to compose the page with ExtJS directly on small screens instead of overriding the ExtJS functionality by CSS.

### Related issue(s)/PR(s)
None known.
